engines/bastion_katello/package.json

```diff
@@ -2,6 +2,24 @@
   "name": "bastion_katello",
   "version": "1.0.0",
   "dependencies": {
-    "bastion": "file:../bastion"
+    "bastion": "file:../bastion",
+    "eslint": "~0.14.1",
+    "eslint-plugin-angular": "0.0.3",
+    "generator-bastion": "~0.1.2",
+    "grunt": "~0.4.5",
+    "grunt-angular-gettext": "~0.2.15",
+    "grunt-bower-task": "yatskevich/grunt-bower-task#v0.5.0",
+    "grunt-concurrent": "~1.0.0",
+    "grunt-eslint": "~6.0.0",
+    "grunt-htmlhint": "~0.4.1",
+    "grunt-karma": "^1.0.0",
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.22",
+    "karma-coverage": "^1.0.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-ng-html2js-preprocessor": "^1.0.0",
+    "karma-phantomjs-launcher": "^1.0.0",
+    "require-dir": "~1.2.0",
+    "phantomjs-prebuilt": "2.1.10"
```

Comment:
Re-defining all of these here is kinda ugly, but this code rarely changes so I think it's OK. Something in updating to NodeJS 10+ seems to have affected how all the relative imports work with Grunt.